server/grails-app/services/org/gokb/PackageSourceUpdateService.groovy

```diff
@@ -79,13 +79,18 @@ class PackageSourceUpdateService {
       title_ns_serial_id = pkg_source?.titleIdSerial?.id ?: null
       title_ns_mono_id = pkg_source?.titleIdMonograph?.id ?: null
 
+      if ( restrictSize ) {
+        def ignoreSizeLimit = pkg_source.getIgnoreSizeLimit()
```

Comment:
Not sure how it would happen for there to be no source, but for safeties sake we should do `pkg_source?.ignoreSizeLimit`here.